src/somef/export/json_export.py

```diff
@@ -152,8 +151,10 @@ def format_date(date_string):
         codemeta_output["programmingLanguage"] = [x[constants.PROP_RESULT][constants.PROP_VALUE] for x in repo_data[constants.CAT_PROGRAMMING_LANGUAGES]]
     if constants.CAT_REQUIREMENTS in repo_data:
         codemeta_output["softwareRequirements"] = [x[constants.PROP_RESULT][constants.PROP_VALUE] for x in repo_data[constants.CAT_REQUIREMENTS]]
+    if constants.CAT_CONTINUOS_INTEGRATION in repo_data:
+        codemeta_output["continuousIntegration"] = repo_data[constants.CAT_CONTINUOS_INTEGRATION][0][constants.PROP_RESULT][constants.PROP_VALUE]
     if constants.CAT_WORKFLOWS in repo_data:
-        codemeta_output["continuousIntegration"] = repo_data[constants.CAT_WORKFLOWS][0][constants.PROP_RESULT][constants.PROP_VALUE]
+        codemeta_output["workflows"] = repo_data[constants.CAT_WORKFLOWS][0][constants.PROP_RESULT][constants.PROP_VALUE] 
```

Comment:
There is no category "workflows" in codemeta nor schema.org. So this should not be there.